buildspec.yml

```diff
@@ -0,0 +1,22 @@
+version: 0.2         
+phases:
+  install:
+    runtime-versions:
+      golang: 1.12
+  pre_build:
+    commands:
+      - echo Building the AWS for Fluent Bit image
+  build:
+    commands:
+      # Command to build your project
+      - make release
+
+      # List the docker images 
+      - docker images
+
+      # Push the image to ECR in the same account and same region the pipeline is hosted.
+      # `amazon/aws-for-fluent-bit` is the image name, and `latest` is the tag name.
+      - ecs-cli push amazon/aws-for-fluent-bit:latest
```

Comment:
this will create an ECR repo named "{account ID}.ecr.{region}.amazonaws.com/amazon/aws-for-fluent-bit"

This is really similar to the name of the public repo... and the pipeline runs in the same account that hosts the images. 

Can this temporary repo used by the pipeline have a different name that clearly indicates its purpose?